src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+for (let i = 0; i < rows.length; i++) {
+  rows[i].children[3].after(rows[i].children[1].cloneNode(true));
```

Comment:
```suggestion
  rows[i].lastElementChild.before(rows[i].children[1].cloneNode(true));
```
Або так, як варіант (це якщо, раптом, змінилась би кількість колонок, а нам все одно потрбіно було б додавати перед передостанньою)